public/css/main.css

```diff
@@ -118,6 +119,62 @@ body {
 
 /*** EVENT DETAILS ***/
 
+.event-details {
+  text-align: center;
+}
+
+.event-img {
+  width: 100%;
+  margin-bottom: 2rem;
+}
+
+.event-name {
+  color: #5C5C5C;
+  font: bolder 2.5rem 'Lato', sans-serif;
+  margin-bottom: .4rem;
+}
+
+.description {
+  margin-top: 2rem;
+}
+
+.cost {
+  margin-top: 5rem;
+  margin-bottom: 5rem;
```

Comment:
So at the moment each thing on this page has a separate margin value - I think it would be better if each separate 'block' just had the same `margin-bottom` value - maybe even make it a separate class. 

So the image block, name+date block, price block, map block - they should all have the same margin-bottom for visual consistency.